history/report.py

```diff
@@ -244,6 +244,27 @@ def report():
         template_args["puf_combined_table"] = None
         template_args["puf_income_table"] = None
         template_args["puf_payroll_table"] = None
+        template_args["puf_salaries_and_wages_table"] = None
```

Comment:
This looks like it does what you need, but there's probably an easy way use few lines of code by, e.g., iterating over the dict and assigning `None` to all arguments and then just updating the value for the key `puf_msg`.